lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
Absolutely. On a related side-note, do you want comments in code wrapped tighter than what I've been doing? It seemed that a lot of the existing comments are wrapped tighter, but ESLint is using 100 characters for the line-length. I personally got used to 132 back in the dark ages of print but accommodate whatever.